.gitlab/pipeline/test.yml

```diff
@@ -45,6 +45,22 @@ test-linux-oldkernel-stable:
   tags:
     - oldkernel-vm
 
+# https://github.com/paritytech/ci_cd/issues/864
+test-linux-stable-runtime-benchmarks:
+  stage: test
+  extends:
+    - .docker-env
+    - .common-refs
+    - .run-immediately
+    - .pipeline-stopper-artifacts
+  variables:
+    RUST_TOOLCHAIN: stable
+    # Enable debug assertions since we are running optimized builds for testing
+    # but still want to have debug assertions.
+    RUSTFLAGS: "-Cdebug-assertions=y -Dwarnings"
+  script:
+    - cargo nextest run --features runtime-benchmarks benchmarking
```

Comment:
```suggestion
    - cargo nextest run --features runtime-benchmarks benchmarking --locked -q --profile testnet
```

Reply:
All the benchmark tests are under modules called `benchmark::` so if we remove the 'ing' off the end then we should catch all of the autogenerated ones.